broker/config/settings.yml

```diff
@@ -166,6 +166,7 @@ defaults: &defaults
       "deployment.servicefabrik.io_v1alpha1_directors.yaml": <%=base64_template('crds/deployment.servicefabrik.io_v1alpha1_directors.yaml') %>
       "lock.servicefabrik.io_v1alpha1_deploymentlocks.yaml": <%= base64_template('crds/lock.servicefabrik.io_v1alpha1_deploymentlocks.yaml') %>
       "backup.servicefabrik.io_v1alpha1_defaultrestores.yaml": <%= base64_template('crds/backup.servicefabrik.io_v1alpha1_defaultrestores.yaml') %>
+      "backup.servicefabrik.io_v1alpha1_defaultboshrestores.yaml": <%= base64_template('crds/backup.servicefabrik.io_v1alpha1_defaultboshrestores.yaml') %>
```

Comment:
along with this, add the `plan.manager.resource_mappings` parameter as well.

Reply:
We decided to keep the bosh based restore dormant till the appropriate ITs are added (see https://github.infra.hana.ondemand.com/cloudfoundry/service-fabrik-product-component/pull/1713). Hence will modify this also in next PR.